.eslintrc.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
       'rules': {
         '@o3r/json-dependency-versions-harmonize': ['error', {
           ignoredPackages: ['@o3r/build-helpers'],
+          ignoredDependencies: ['npm', 'yarn'],
```

Comment:
do we have `npm` and `yarn` dependencies ?

Reply:
in `engines` properties, yes

Reply:
The story is more complicated:
- The `create` packages requires `yarn < 2.0.0`
- The workspace package.json explicitly forbid `npm` when some other packages are restriction on it.

The only compromise that can be found is:
- ignoring only `npm` at workspace level (instead of `npm` and `yarn`)
- override the rule regarding  the ignore`yarn` in `create` packages only
- remove the `yarn` and `npm` engines restriction in `showcase-sdk`